lib/manageiq/api/client/collection.rb

```diff
@@ -0,0 +1,52 @@
+require_relative "mixins/action_mixin"
```

Comment:
I think in gems, the pattern is to prefer requiring from lib... so `require 'manageiq/api/client/mixins/action_mixin'`